targets/jm-XAC-input-34-1.0A/board.h

```diff
@@ -26,5 +26,3 @@
 #define UART_PIN PA_2
 #define UART_PIN_AF LL_GPIO_AF_1
 #define USART_IDX 2
-
-#define JD_CONFIG_APP_PROCESS_HOOK  1
```

Comment:
I guess app process is weak defined?

Reply:
that's correct

Reply:
I actually checked disassembly of xac f/w and it has the correct version